aiosmtpd/docs/controller.rst

```diff
@@ -40,7 +40,9 @@ to the console.  Start by implementing a handler as follows::
     ...         print('Message from %s' % envelope.mail_from)
     ...         print('Message for %s' % envelope.rcpt_tos)
     ...         print('Message data:\n')
-    ...         print(envelope.content.decode('utf8', errors='replace'))
+    ...         for ln in envelope.content.decode('utf8', errors='replace').splitlines():
+    ...             print(f'> {ln}'.strip())
+    ...         print()
```

Comment:
Some invisible spaces causes pytest's doctest to fail; I have to split the lines and strip() them.